python/vimsence.py

```diff
@@ -33,12 +36,11 @@
 if vim.eval('exists("g:vimsence_client_id")') == '1':
     client_id = vim.eval('g:vimsence_client_id')
 
+with open(os.path.join(vim.eval('s:plugin_root_dir'), '..', 'vimsence.json'), 'r') as config_file:
+    config = json.load(config_file)
+
 # Contains which files has thumbnails.
```

Comment:
Could you update the comment to explain that we're retrieving the data from the configuration file?